napari/layers/shapes/_shapes_mouse_bindings.py

```diff
@@ -198,10 +207,19 @@ def add_path_polygon(layer, event):
         layer._moving_value = copy(layer._value)
         layer._is_creating = True
         layer._set_highlight()
+    elif event.type == 'mouse_press' and layer._mode == Mode.ADD_POLYGON_LASSO:
+        index = layer._moving_value[0]
+        vertices = layer._data_view.shapes[index].data
+        vertices = rdp(vertices, epsilon=0.5)
```

Comment:
One point of discussion is whether we should expose this epsilon parameter to the user somehow.

Reply:
Not sure I would expose, but I would make some empirical tests of drawing curves slower or faster and with more or less curvature/sharp corners/wiggling, and empirically find a reasonable epsilon.

Reply:
I did that for regular cell shapes. However, the choice of this parameter does depend on what you are trying to annotate and how accurate you need it to be.